go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/alexeldeib/skewer
+
+go 1.13
+
+require (
+	github.com/Azure/azure-sdk-for-go v38.1.0+incompatible
```

Comment:
that's kind of old, I think

Reply:
related to issue with dependencies in RP. will try to upgrade back to v45 and see if I can't resolve it.